travis.sh

```diff
@@ -12,4 +12,4 @@ cmake -Dgtest_build_samples=ON \
       -DCMAKE_CXX_FLAGS=$CXX_FLAGS \
       ../../$GTEST_TARGET
 make
-make test
+CTEST_OUTPUT_ON_FAILURE=1 make test
```

Comment:
Output anything outputted by the test program if the test should fail. 

That environment variable will show test output on Travis to simplify debugging failed tests